src/Components/App/App.js

```diff
@@ -1,25 +1,33 @@
-import React, { useState } from 'react';
+import React, { Suspense, useEffect, useRef, useState } from 'react';
 import { TopSection } from '../TopSection';
 import { Sprite } from '../Sprite';
 import { AvailableHotelsContext } from '../../Context/AvailableHotelsContext';
 import { AvailableHotels } from '../AvailableHotels/AvailableHotels';
 import { Homes } from '../Homes';
+import { Advantages } from '../Advantages';
 
 import './App.css';
+import { Loader } from '../Loader/Loader';
 
 export const App = () => {
   const [available, setAvailable] = useState([]);
+  const availableRef = useRef(null);
+
+  useEffect(() => {
+    available.length > 0 && availableRef.current.scrollIntoView({ behavior: 'smooth' });
+  }, [available.length]);
+
   return (
     <>
       <Sprite />
       <AvailableHotelsContext.Provider value={{ available, setAvailable }}>
         <TopSection />
-        {available.length > 0 && <AvailableHotels />}
-        
-
+        <Suspense fallback={<Loader />}>
```

Comment:
If you want to use SUspense don't forget, tat your component inside must be lazy